digital_land/phase/lookup.py

```diff
@@ -59,6 +68,9 @@ def process(self, stream):
                         # or by the CURIE
                         or self.lookup(prefix=prefix, reference=reference)
                     )
+                if self.redirect_lookups:
```

Comment:
We need to factorise across LookupPhase and PrintLookupPhase to stop them separating in the future. currently they're using different code to do the same thing which they shouldn't be. Specifically PrintLookupPhase could be another child of the Lookup phase class